documentation/leo/03_language.md

```diff
@@ -412,23 +432,46 @@ program test.aleo {
 ```
 
 #### Mapping
-The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the following functions.
+
+The mapping struct allows the programmer to apply updates to a program mapping data structure by calling one of the
+following functions.
 
 #### get
+
 A get command, e.g. `current_value = Mapping::get(counter, addr);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`
 If the value at `addr` does not exist, then the program will fail to execute.
 
 #### get_or_init
+
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
-If the key is not present, `0u64`  is stored in `counter` and stored in `current_value`.
+If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
+
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
+## For Loops
+
+For Loops are declared as `for {variable: type} in {lower bound: type}..{upper bound: type}`. Unsigned integer 
```

Comment:
```suggestion
For Loops are declared as `for {variable: type} in {lower bound}..{upper bound}`. Unsigned integer 
```
The lower and upper bounds are literals - their type is specified with the value `1u8`, `1337u16`
There is no colon required @moAlobaidi